src/AppBundle/Resources/translations/common.en.yml

```diff
@@ -26,6 +27,7 @@ manage: Manage
 annualDeputyReport: Annual deputy report
 currentDeputyReport: Current deputy report
 submittedReports: Submitted reports
+unsubmittedReport: Incomplete deputy report
```

Comment:
hmmm key is a bit confusing here

Reply:
in the backend, incomplete means a report not submitted, but this new functionality calls them "incomplete" in the UI. I and Biggs decided to keep the naming consistent